m4_sessions/1706-inning/group_four.md

```diff
@@ -2,8 +2,62 @@
 
 ### Session Title:
 
+"Getting involved and giving back as a beginner"  
+
 ### Session Outcomes: 
 
+1. Doc to add to their portfolio
+2. Interest/list of OSS to make first PR
+3. Check list to work through for first PR (fulfills PD deliverables, including [blog post (all mods), PR (mod 4)])
+
+Feelings and thoughts:
+
+- people feel confident in their ability to contribute to the community
+- people have the knowledge of where to go to contribute
+- the knowledge of HOW to contribute (like OSS PR)
+
+
+
+
+Ideas:
+
+- videos
+- list of projects to contribute to
+- how to make an impact at Turing
+  - make PR on lessons
+  - make oss contributions
+  - PR checklist/template
+
 ### Session Outline:
 
+
+- do we have a responsibility to "give back" to the community?
+
+- what does "giving back" look like?
```

Comment:
And maybe even ask the question...who can giving back impact the most?  Or who do you want to make an impact on when you "give back"?